dist/build.gradle

```diff
@@ -111,7 +111,9 @@ task docker(group: 'Build',
             dependsOn: tasks.distDir,
             type: DockerBuildImage) {
     inputDir = file('.')
-    tag = "line/centraldogma:${project.version}"
+    def name = 'line/centraldogma:'
+    tags = [name + "${project.version}",
+            name + "latest"]
```

Comment:
- I'm curious how Docker handles snapshot versions. Should we replace 'SNAPSHOT' with a timestamp?
- How do we prevent users from fetching unreleased snapshot versions?